Robert Chaney

rwchaneyjr opened the pull request:
this is a cutscene from the main into a scene called WolfPistol for 10 or 15 seconds and then back to the main scene

rwchaneyjr committed:
Robert Chaney

 this is a cutscene from the main into a scene called WolfPistol for 10 or 15 seconds and then back to the main scene

rwchaneyjr committed:
Robert Chaney

a cut scene with 3 sprites

CaseyWalshWarder commented:
Remove the added package(s). This is open source, and we also don't want to make the project larger for no reason.

The images shown in the slideshow don't matter right now, so just use anything already in the project.

rwchaneyjr committed:
Robert Chaney

cut scene in skycore with images which are timed. Image  and sprite arrays

CaseyWalshWarder commented:
Remove the recorder package, and everything in the Wolf and pistol sprites folder.

In Move.cs, we need it to work regardless of the length of the array of sprites. You could do something like this in a coroutine:

private void IEnumerator()
{
  Image image = GetComponent<Image>();
  for (int i = 0; i < S.Length; i++)
  {
      image.sprite = S[i];
      yield return new WaitForSeconds(3);
  }
  // code to load the main scene here
}

rwchaneyjr committed:
Robert Chaney

updated slide show for cut sceene

CaseyWalshWarder commented:
The most recent commit does not address the fixed number of images and it still includes third party art.

rwchaneyjr committed:
Robert Chaney

Start Scene to cutscene   3 images back to main scene. all the sprites were in the project already. There is an array for the images and the sprites. One references the images and it can be adapted to fit the size of images and sprites

CaseyWalshWarder commented:
I have to close this pull request. I've tried to explain these issues multiple times.

- It adds an unnecessary package. The main issue is you would need to check the license for that package and make sure we can distribute it.
- It adds images which we didn't make. Someone else has the copyright to them, so we cannot include them in an open source. They also won't be kept permanently because this is for a slideshow whose images will show the story.
- The code will not work with any number of images. Maybe it works with 3 for example, but it won't work if the slideshow ends up being 10 images long.
